app/views/users/index.html.erb

```diff
@@ -20,7 +20,7 @@
   </div>
 </div>
 
-<div class="row to-member-cards">
+<div class="card-container to-member-cards">
```

Comment:
instead of creating `card-container` class you could have given some styles to `to-member-cards`